image/scanner/scripts/import-additional-cas

```diff
@@ -13,4 +16,7 @@ copy_existing () {
 
 copy_existing /usr/local/share/ca-certificates
 
-update-ca-trust
+# Copy the custom trusted CA bundles injected by the Openshift Network Operator.
+copy_existing /etc/pki/injected-ca-trust
+
+update-ca-trust extract
```

Comment:
what does `extract` do? I don't see it used here https://github.com/stackrox/stackrox/blob/master/image/static-bin/import-additional-cas

Reply:
Ah ok https://www.unix.com/man-page/centos/8/update-ca-trust/ tells me it's the same but with more warnings. What encouraged the switch?

Reply:
I see you answered all of this in the description I clearly didn't read